doc/specifications/drafts/content_edit.md

```diff
@@ -0,0 +1,29 @@
+# Symfony forms content edit
+
+## Synopsis
+repository-forms currently supports creating content without an intermediate draft.
+
+Support must be added for editing content. It implies creating a new version of an existing content,
+rendering a form with the previous version's fields values, and allowing the user to save the draft,
+cancel or publish the new version. It should also allow choosing a language, either for translation
+or 
+
+Ideally, the draft would only be created when we need it, not on the first display.
```

Comment:
> This is mainly to remove a potential DOS attack and also create many "loose drafts" whenever users are e.g. registering. Can easily be exploited to fill up the DB.

Creating a new (v0) content without creating a draft is already possible.
We had decided to shape it that way to limit the risk of the DB being filled up.

> Edits of existing versions should create a draft as it is needed for checking if it is already being worked on etc. as AR mentions.
> The overal idée is to not create a draft until you actually publish something.

These two sentences are contradicting each other, I'm afraid.
We can create the draft when a content edit (version > 1) is displayed, or try to do it later.
For now, the Public API and REST API don't do anything in this regard. PlatformUI has its own logic for it, a bit like what Legacy was doing, but afaik it doesn't prevent a user from creating multiple drafts of the same content.

Is version / conflict management something we want to integrate into the current spikes or not ?